RELEASE_NOTES.md

```diff
@@ -2,7 +2,9 @@
 
 Changes:
 
+  * Added back metrics for Illumina Tests
   * Added test data for NovaSeq X
+  * Update tests to include sequencing kits
```

Comment:
Add a note about the Illumina metrics number format change. This will make no difference to some, including us. Others may currently have a different format, and may be depending on that.